src/java/org/apache/cassandra/service/accord/AccordCommandStore.java

```diff
@@ -94,17 +120,67 @@ public AccordCommandStore(int id,
         this.commandCache = stateCache.instance(TxnId.class, accord.local.Command.class, AccordSafeCommand::new, AccordObjectSizes::command);
         this.commandsForKeyCache = stateCache.instance(RoutableKey.class, CommandsForKey.class, AccordSafeCommandsForKey::new, AccordObjectSizes::commandsForKey);
         executor.execute(() -> CommandStore.register(this));
+        executor.execute(this::loadRangesToCommands);
     }
 
-    @Override
-    public boolean inStore()
+    private void loadRangesToCommands()
     {
-        return Thread.currentThread().getId() == threadId;
+        AsyncPromise<CommandsForRanges> future = new AsyncPromise<>();
+        AccordKeyspace.findAllCommandsByDomain(id, Routable.Domain.Range, ImmutableSet.of("txn_id", "status", "txn", "execute_at", "dependencies"), new Observable<UntypedResultSet.Row>()
+        {
+            private CommandsForRanges.Builder builder = new CommandsForRanges.Builder();
+            @Override
+            public void onNext(UntypedResultSet.Row row) throws Exception
+            {
+                TxnId txnId = AccordKeyspace.deserializeTxnId(row);
```

Comment:
just thought about this... the listener isn't durable, so I might need to load them to re-create the listeners... :sigh: